src/services/wallets/torus/tweb3.ts

```diff
@@ -28,8 +28,10 @@ const popupStyling = {
       torusGray2: "#FBF7F3",
     },
   },
-  logoDark: "https://www.projectliberty.io/assets/images/brand-icon.png",
-  logoLight: "https://www.projectliberty.io/assets/images/brand-icon.png",
+  logoDark:
+    "https://www.pngkit.com/png/full/119-1193080_technology-electronics-icons-png.png",
+  logoLight:
+    "https://www.pngkit.com/png/full/119-1193080_technology-electronics-icons-png.png",
```

Comment:
These urls feel weird. Is that really what we want?

Reply:
Don't think it will be long term but we needed something. If we have a link to an unfinished or liberty icon I can use I'll put that in.

Reply:
@claireolmstead do you have something that could be used here?

Reply:
I replaced it with the public github logo. It's a bit small but it's better and ours.